firestore-tos-extension/POSTINSTALL.md

```diff
@@ -0,0 +1,83 @@
+### See it in action
+
+You can test out this extension right away!
+
+1. Go to your [Authentication dashboard](https://console.firebase.google.com/project/${param:PROJECT_ID}/authentication/users) in the Firebase console.
+
+1. Click **Add User** to add a test user, then copy the test user's UID to your clipboard.
+
+2. Access the user through any of the authetication methods.
+
+```js
+    const user = await getAuth().getUser(uid);
```

Comment:
This needs some more context - admin SDK? Imports etc

Reply:
Ok, will add more details on examples 👍🏻

Reply:
Updated with <v9 and v9 versions.